### PR TITLE
Add new Cask : SUIDGuardNG 1.0.6

### PR DESCRIPTION
**Pull request by iiharu**
##### Instructions
- Look for and complete the section relevant to your submission. Delete the others, including these `Instructions`.
- `{{cask_file}}` represents the cask file you’re submitting/editing (if applicable).
- If there’s a checkbox you can’t complete for any reason, that’s OK. Just explain in detail why you weren’t able to do so.
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by yasu100** [comment]
I think that test failed because the installation on OS X 10.11 is prohibited.

Installing this cask on OS X 10.11 makes SUIDGuard crash on boot.

File: Casks/suidguardng.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  sha256 '47fcfbf2278cf23da46cb56f07b98647dc21b24d4e574a56713da371b688f7cd'
+
+  url 'https://www.suidguard.com/downloads/SUIDGuardNG-106.pkg'
+  name 'SUIDGuardNG'
+  homepage 'https://www.suidguard.com/'
+  license :bsd
+
+  depends_on macos: '<= :yosemite'
+
+  pkg 'SUIDGuardNG-106.pkg'
```
Comment (vitorgalvao) [inline Casks/suidguardng.rb]: `pkg "SUIDGuardNG-#{version.no_dots}.pkg"`

File: Casks/suidguardng.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+cask 'suidguardng' do
+  version '1.0.6'
+  sha256 '47fcfbf2278cf23da46cb56f07b98647dc21b24d4e574a56713da371b688f7cd'
+
+  url 'https://www.suidguard.com/downloads/SUIDGuardNG-106.pkg'
```
Comment (vitorgalvao) [inline Casks/suidguardng.rb]: `url "https://www.suidguard.com/downloads/SUIDGuardNG-"{version.no_dots}.pkg"`

**Comment by yasu100** [comment]
Fixed.

**Pull request merged**